ts-apps/decision-logic-visualizer/src/lib/displayers/flow/flow.svelte

```diff
@@ -15,9 +14,9 @@ when trying to make ladder diagrams. -->
   // Check preconditions
   // TODO: Think more about whether we should really require consumers to set the lir root in the future;
   // but for now, it's enough to just check that it's been done.
-  if (env.getLirRegistry().getRoot(context, LADDER_VIZ_ROOT_TYPE) !== node) {
+  if (env.getTopFunDeclLirNode(context) !== node) {
```

Comment:
unrelated Refactor / Fix: Have consumers of the visualizer lib define the LirRootType themselves and pass it into the visualizer via the LadderEnv